File: src/sas/handler/testclient/TestEmptyHandler.java
package sas.handler.testclient;

import org.apache.mina.core.session.IoSession;
import sas.bean.TestEmpty;
import sas.core.BeanHandler;
import sas.core.BeanManager;
import sas.core.Log;

public class TestEmptyHandler extends BeanHandler<TestEmpty>
{
	/*\
	\*/

	@Override
	public void onProcess(BeanManager manager, IoSession session, TestEmpty arg)
	{
		Log.log.debug("{}.onProcess: arg={}", getClass().getName(), arg);
	}
}

File: port/csharp/jane/handler/testclient/TestEmptyHandler.cs
using Jane.Bean;

namespace Jane.handler.testclient
{
	public class TestEmptyHandler
	{
		/*\
		\*/

		public static void onProcess(NetManager manager, IBean _arg_)
		{
			TestEmpty arg = (TestEmpty)_arg_;
			System.Console.WriteLine("{0}.onProcess: arg={1}", arg.GetType().Name, arg);
		}
	}
}
